content/en/bits_ai/bits_ai_sre/investigate_alerts.md

```diff
@@ -21,7 +21,14 @@ There are a few ways to enable Bits for automated investigations:
 
 You can also add the tag to your desired monitors using the Datadog API or Terraform.
 
-An investigation initiates when a monitor transitions to the alert state. Transitions to the warn or no data state, [renotifications][12], and test notifications do not trigger investigations. Additionally, noisy monitors are automatically rate-limited to avoid unnecessary investigations and protect your budget.
+An investigation initiates when a monitor transitions to the alert state. Transitions to the warn or no data state, [renotifications][12], and test notifications do not trigger investigations.
+
+#### Rate limits
+
+|  Type         | Description                                                                                   | Limit                                      |
+|--------------------|----------------------------------------------------------------------------------------------|--------------------------------------------|
+| Monitor limit     | A monitor can trigger up to 2 investigations every 30 minutes.<br>If a monitor is "noisy"&mdash;  that is, if it triggers alerts on 12 or more of the last 14 days, or fires in more than 15% of an hour interval&mdash; it is automatically rate-limited. | 2 executions per 30 minutes per monitor     |
```

Comment:
```suggestion
| Monitor limit     | A monitor can trigger up to two investigations every 30 minutes.<br>If a monitor is "noisy"---that is, if it triggers alerts on 12 or more of the last 14 days, or stays in alert state for more than 15% of an hour interval---it is automatically rate-limited. | 2 executions per 30 minutes per monitor     |
```

Reply:
Updating the em dashes here to markdown, and made a spacing adjustment per the [documentation style guide](https://github.com/DataDog/documentation/blob/master/CONTRIBUTING.md#dashes).

Also, I _think_ `fires in more than 15% of an hour interval` means that the monitor was in alert state for that amount of time, but let me know if that's not the case